lib/bugzilla_conn.py

```diff
@@ -0,0 +1,31 @@
+#! /usr/bin/env python3
+
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+import os
+
+import bugzilla
+
+
+class BugzillaAPIClient:
+
+    @property
+    def URL(self) -> str:
+        return "bugzilla.mozilla.org"
```

Comment:
On final PR, make sure to replace domains with a constant that lives in `constants.py`